### PR TITLE
Feature/hide consolidate utxos

### PR DESCRIPTION
**Pull request by saurabh391**
#### What kind of PR is this?:
Hide consolidate UTXOs option #469

/kind feature

#### What this PR does / why we need it:
This is PR is for the new settings design and countdown management feature.

#### Which issue(s) does this PR fixes?:
Hide consolidate UTXOs option #469

Fixes #

### CODE REVIEW

**Comment by thedoublejay** [comment]
Let's include this on next release

**Comment by izzycsy** [comment]
210317 v2 3 2 MN Settings - Blank

Hi @saurabh391 , settings page is blank. 
[Marvel ref](urlhttps://marvel-live.freetls.fastly.net/serve/2021/2/90c93752e834dcfbb7975bbe48d9c52.png?qual[…]y=95&fake=.png&width=320&crop=&dpr=1.7999999523162842)

**Comment by saurabh391** [comment]
> 210317 v2 3 2 MN Settings - Blank
> 
> Hi @saurabh391 , settings page is blank.
> Marvel ref
I have hide security tab can you please test now @izzycsy

File: webapp/src/containers/SettingsPage/components/SettingsTabHeader/index.tsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ const SettingsTabsHeader = (props: SettingsTabsHeaderProps) => {
             {I18n.t('containers.settings.general')}
           </NavLink>
         </NavItem>
-        {displaySecurityTab && (
+        {/* {displaySecurityTab && (
```
Comment (thedoublejay) [inline webapp/src/containers/SettingsPage/components/SettingsTabHeader/index.tsx:39]: why was this removed?

File: webapp/src/containers/SettingsPage/index.tsx [hunk 2]
```diff
@@ -329,10 +348,13 @@ const SettingsPage: React.FunctionComponent<SettingsPageProps> = (
             handleDropDowns={handleDropDowns}
             handeReindexToggle={handeReindexToggle}
             handeRefreshUtxosToggle={handeRefreshUtxosToggle}
+            sendCountdown={sendCountdown}
+            handeDeletePeersClick={handeDeletePeersClick}
           />
-          {props.isWalletCreatedFlag && props.isWalletEncrypted && (
```
Comment (thedoublejay) [inline webapp/src/containers/SettingsPage/index.tsx:333]: same

File: webapp/src/containers/SettingsPage/saga.tsx [hunk 3]
```diff
@@ -138,6 +138,13 @@ export function* updateSettings(action) {
       if (data.refreshUtxosAfterSaving) {
         yield call(refreshUtxosAfterSavingData);
       }
+      if (
```
Comment (thedoublejay) [inline webapp/src/containers/SettingsPage/saga.tsx:141]: I suggest we use wallet_map.json instead of Persistent Store

**Review by thedoublejay — CHANGES_REQUESTED** [review]

**Comment by thedoublejay** [comment]
@saurabh391  why did you comment out security tab?

**Comment by saurabh391** [comment]
> @saurabh391 why did you comment out the security tab?

@monstrobishi asked me to hide the security tab for now.

**Comment by thedoublejay** [comment]
> > @saurabh391 why did you comment out the security tab?
> 
> @monstrobishi asked me to hide the security tab for now.

@monstrobishi Any reason why? It's the only place where you can change passphrase and lock timeout.

**Comment by monstrobishi** [comment]
I was under the impression it hasn't been implemented yet?

**Comment by thedoublejay** [comment]
> I was under the impression it hasn't been implemented yet?

Oh it's already implemented now. Users can change passphrase and update the auto lock

**Comment by monstrobishi** [comment]
Ok right then we should keep it @saurabh391 , I misunderstood your question that time you asked, I thought it's a new feature

**Comment by saurabh391** [comment]
> > I was under the impression it hasn't been implemented yet?
> 
> Oh it's already implemented now. Users can change passphrase and update the auto lock

I was facing some issues with already implemented logic can you please check.

**Comment by saurabh391** [comment]
> Ok right then we should keep it @saurabh391 , I misunderstood your question that time you asked, I thought it's a new feature

No worries

**Comment by thedoublejay** [comment]
> > > I was under the impression it hasn't been implemented yet?
> > 
> > 
> > Oh it's already implemented now. Users can change passphrase and update the auto lock
> 
> I was facing some issues with already implemented logic can you please check.

What issue?

**Comment by saurabh391** [comment]
> > > > I was under the impression it hasn't been implemented yet?
> > > 
> > > 
> > > Oh it's already implemented now. Users can change passphrase and update the auto-lock
> > 
> > 
> > I was facing some issues with already implemented logic can you please check.
> 
> What issue? pm

I have made the changes and unhide the tab.

**Review by thedoublejay — APPROVED** [review]

**Pull request merged**